### ISSUE TITLE
Missing localization for the charts labels

### ISSUE THREAD

**Issue by fabriziocacicia** [guydavis/machinaris#631]
**Describe the bug**
The charts labels are hardcoded in English.

**To Reproduce**

For example, going to `/web/templates/views/index_script_block.js`:

```js
  text: 'Time - Recent',
    ...

    ...
  text: "Time Taken (seconds)",
```

**Expected behavior**
They should be extracted to `messages.po` and referenced from there.

</details>

**Additional context & screenshots**
<img width="1172" alt="Screenshot 2022-04-03 at 20 04 36" src="https://user-images.githubusercontent.com/7525888/161441797-9edb9f6d-76ae-40c6-8d18-e1ad946c5837.png">

**Comment by guydavis** [comment]
Thanks!  Fixed:

**Comment by fabriziocacicia** [comment]
@guydavis They were not missing only for the charts on the index page. But also in all the other charts, like the ones in `plotting/jobs`, in `plotting/workers`, in `worker`...

**Comment by fabriziocacicia** [comment]
<img width="1154" alt="Screenshot 2022-04-04 at 01 11 13" src="https://user-images.githubusercontent.com/7525888/161452876-69d430a2-91df-4c52-bd35-08866329fec5.png">

For example the `Plotting` chart in the `worker` page is completely un-localized.

**Comment by guydavis** [comment]
Thanks! Fixed:
![image](https://user-images.githubusercontent.com/12936/161454816-2cf17d4c-5712-4788-a3e8-13d46a9c04d5.png)

**Comment by guydavis** [comment]
This fix is now available in the [Machinaris v0.7.2 release](https://github.com/guydavis/machinaris/releases/tag/0.7.2) today. Thanks!
